README.md

```diff
@@ -92,6 +92,11 @@ docker run -it \
     -e FLOW_ACCOUNT_PUBLIC_KEY=519e9fbf966c6589fafe60903c0da5f55c5cb50aee5d870f097b35dfb6de13c170718cd92f50811cdd9290e51c2766440b696e0423a5031ae482cca79e3c479 \       -e FLOW_INIT_ACCOUNTS=0 \
     -e FLOW_ACCOUNT_ADDRESS=0xf8d6e0586b0a20c7 \
     -e FLOW_AVATAR_URL=https://avatars.onflow.org/avatar/ \
+    -e CONTRACT_FUNGIBLE_TOKEN=0xee82856bf20e2aa6 \
+    -e CONTRACT_FLOW_TOKEN=0x0ae53cb6e3f42a79 \
+    -e CONTRACT_FUSD=0xf8d6e0586b0a20c7 \
+    -e TOKEN_AMOUNT_FLOW=100.0 \
+    -e TOKEN_AMOUNT_FUSD=100.0 \
```

Comment:
is this change related to PR https://github.com/onflow/kitty-items/pull/191#pullrequestreview-852118704 ? if so, similar thought: should we initialize the token and FUSD to a much bigger value (say 10,000) so that developers never need to worry about adding funds to the account during test/development?

Reply:
Leaving these to 100.0 as discussed in https://github.com/onflow/kitty-items/pull/191#pullrequestreview-852118704